source/agora/node/Validator.d

```diff
@@ -200,6 +205,13 @@ public class Validator : FullNode, API
             log.fatal("rebuildQuorumConfig: Exception thrown: {}", e);
             assert(0);
         }
+
+        foreach (hash; keys.filter!(key => key != this_enroll))
+        {
+            UTXO utxo;
+            assert(this.utxo_set.peekUTXO(hash, utxo));
+            this.required_peer_utxos[hash] = utxo;
+        }
```

Comment:
This should probably be in `buildRequiredKeys` instead (and the removed call be reinstated).

Reply:
Right, I will reinstate it.